src/hooks/useSwapCallback.ts

```diff
@@ -75,7 +75,7 @@ function useSwapCallArguments(
             feeOnTransfer: false,
             allowedSlippage: new Percent(JSBI.BigInt(allowedSlippage), BIPS_BASE),
             recipient,
-            deadline: deadline.toNumber()
+            ttl: deadline.toNumber()
```

Comment:
Any idea why this change on xDAI?

Reply:
https://github.com/Uniswap/uniswap-sdk/blob/14aeb6eda9992e029866d0e3467bd3d0bba2b25a/src/router.ts

fork likely using an older version